layouts/_default/baseof.html

```diff
@@ -1,6 +1,91 @@
 <head>
+    <script type="text/javascript" src="https://www.datadoghq-browser-agent.com/datadog-logs-us.js"></script>
+    <script>
+      // Set your Public API key
+      DD_LOGS.init({
+        publicApiKey: 'pubf839012dcb041ba2b4478770b9e96d11',
+        forwardErrorsToLogs: true,
+      });
+
+      DD_LOGS.addLoggerGlobalContext('env', '{{ $.Site.Params.environment }}');
+      DD_LOGS.addLoggerGlobalContext('service', 'docs');
+      DD_LOGS.addLoggerGlobalContext('host', window.location.host);
+      window.addEventListener('load', function () {
+        let pageNav = performance.getEntriesByType("navigation")[0];
+        let pageNavStats = {};
+        try {
+          // Measuring DNS lookup time
+          pageNavStats.dnsTime = pageNav.domainLookupEnd - pageNav.domainLookupStart;
+          // Quantifying total connection time
+          pageNavStats.connectionTime = pageNav.connectEnd - pageNav.connectStart;
+          pageNavStats.tlsTime = 0; // <-- Assume 0 by default
+
+          // Did any TLS stuff happen?
+          if (pageNav.secureConnectionStart > 0) {
+            // Awesome! Calculate it!
+            pageNavStats.tlsTime = pageNav.connectEnd - pageNav.secureConnectionStart;
+          }
+          // Cache seek plus response time
+          pageNavStats.fetchTime = pageNav.responseEnd - pageNav.fetchStart;
+
+          // Service worker time plus response time
+          pageNavStats.workerTime = 0;
+
+          if (pageNav.workerStart > 0) {
+            pageNavStats.workerTime = pageNav.responseEnd - pageNav.workerStart;
+          }
+          // Request plus response time (network only)
+          pageNavStats.totalTime = pageNav.responseEnd - pageNav.requestStart;
+
+          // Response time only (download)
+          pageNavStats.downloadTime = pageNav.responseEnd - pageNav.responseStart;
+
+          // Time to First Byte (TTFB)
+          pageNavStats.ttfb = pageNav.responseStart - pageNav.requestStart;
+
+          // HTTP header size
+          pageNavStats.headerSize = pageNav.transferSize - pageNav.encodedBodySize;
+
+          // Compression ratio
+          pageNavStats.compressionRatio = pageNav.decodedBodySize / pageNav.encodedBodySize;
+
+          let pageStats = {
+            "location": JSON.parse(JSON.stringify(window.location)),
+            "performance": {
+              "navigation": pageNavStats
+            }
+          };
+          // send the page stats
+          DD_LOGS.logger.info('Page loaded', pageStats);
+        } catch (e) {
+          DD_LOGS.logger.error('Page stats collection error', {'message': e});
+        }
+        try {
+          let pageResources = performance.getEntriesByType("resource");
+          if (pageResources) {
+            pageResources.forEach(el => {
```

Comment:
Can we avoid `forEach` as inline js isn't run through babel and its less compatible with older browsers.